dev-notes/littlefs-problems.md

```diff
@@ -0,0 +1,57 @@
+# Problems with LittleFS
+
+The main problem with LittleFS is its space inefficiency (see LittleFS issue
+[#645](https://github.com/littlefs-project/littlefs/issues/645)).
+
+We are using 64 KiB for LittleFS storage, flash memory on nRF52840 is 1 MiB, but
```

Comment:
@arturkow2000 could you please elaborate that? Can we somehow tell how many fw consume? Maybe split that into libs?

Reply:
@TomaszAIR Described in separate document, generally enabling optimization (`-Oz`) halves binary size. We may save something 64 KiB by optimizing Trussed, getting rid of SW crypto etc, also these numbers may not be accurate. 